tests/unit_tests/test_wave_optional.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+from swap_meet.item import Item
+from swap_meet.vendor import Vendor
+from swap_meet.clothing import Clothing
+from swap_meet.decor import Decor
+from swap_meet.electronics import Electronics
+
+
```

Comment:
Great job adding tests for the extra functionality you did! Excellent job taking on the optional enhancements!